recipes-ni/sysconfig-settings-ssh/sysconfig-settings-ssh_1.0.bb

```diff
@@ -0,0 +1,28 @@
+SUMMARY = "System configuration files for ssh"
```

Comment:
```
 sysconfig-settings-ssh: Add new package

Add sysconfig-settings-ssh package that enables SSH option on MAX
and install it on safemode.
```

Could you reword this commit to clarify that this new (sub)package adds uixml element which populate the 'enable SSH' checkbox in MAX; not that it *enables* SSH.